CinemaStarProjectTemplate/CinemaStarProjectTemplate/Modules/MoviesDetailModule/View/TableView/OriginalCells/CollectionView/SimilarMoviesCollectionViewCell.swift

```diff
@@ -0,0 +1,120 @@
+// SimilarMoviesCollectionViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячейка для показа идентичных фильмов
+final class SimilarMoviesCollectionViewCell: UICollectionViewCell {
+    // MARK: - Visual Components
+
+    lazy var similarMovieStackView: UIStackView = {
```

Comment:
```suggestion
    lazy var similarMovieStackView  = {
```
так на будущее не обязательно писать тип так как ты в замыкании его возвращаешь